### DISCUSSION TITLE
Problem with Qfield

### DISCUSSION THREAD

**Discussion by fantan61** [opengisch/QField#5493] [Bugs]
Dear opengis team,

I have the following problem with the Qfieldcloud:
I can see my project data and images on my cell phone, but I cannot download them (not even via USB). The following error message appears:
![20240730_130722](https://github.com/user-attachments/assets/12f0ed87-655f-4ca1-b033-6d74c858aacf)
![20240730_130741](https://github.com/user-attachments/assets/12ab32a4-d415-4f27-8199-14cc805a32e6)

 would be very grateful if you suggested a solution, because the data is very important to me.

**Comment by meyerlor** [comment]
Quota Error means, you need to upgrade your storage in QField Cloud
